### ISSUE TITLE
[BUG] 1.1.8, 2.x database, "Sort in alphabetical order" non-functional

### ISSUE THREAD

**Issue by ageru** [jobe-m/ownkeepass#108]
# Steps to reproduce
1. Open ownKeepass, got to settings, make sure "Sort in alphabetical order" is deselected. Close ownKeepass.
2. Create a Keepass 2 database in KP2 on Windows. Protect it with a password and keyfile. Create groups and create various entries in a non-alphabetical order. Don't reorder the entries or groups.
3. Transfer database and keyfile to Jolla phone.
4. Open database with ownkeepass.
5. Entries are not shown in alphabetical order as expected.
6. Go to settings, make "Sort in alphabetical order" is selected.
7. Close the database.
8. Re-open the database.

# Actual result
Entries are not shown in alphabetical order

# Expected result
Entries should be shown in alphabetical order

# PS
Hope I'm not jumping the gun again on something you were already working on... :stuck_out_tongue_winking_eye:

**Comment by jobe-m** [comment]
Yes, that's right. Alphabetical sorting of entries for Keepass 2 DB is not yet implemented. I will do this in one of the next releases :)
Thanks for reporting
